docs/installation.rst

```diff
@@ -13,6 +13,22 @@ Requirements
 - The ``--container-runtime`` option must correspond to the containerization tool you use.
 - ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.
 
+Recommended Installation Method
+********************************
+
+.. note::
```

Comment:
make it a note or a header with a paragraph, a header for a note is confusing, looks like someone forgot to populate the text